docs/concepts/experiment/index.md

```diff
@@ -1,10 +1,13 @@
 # Experiments
 
-An 'Experiment' is the current name used in Open Chat Studio to refer to a 'chatbot'. The name may change in the
-future.
+An 'Experiment' is the current name used in Open Chat Studio to refer to a 'chatbot'. This will soon be a legacy term as we transition fully to the term 'Chatbots.'
 
 An Experiment links all the configuration and data for a chatbot including user sessions, data, actions etc.
 
+!!! warning "Deprecation Warnig"
+
+    The term will be phased out as we fully adopt 'Chatbots' instead. Bot creation will shift from the current 'form-based' method to primarily using the pipeline approach. All existing experiments will be smoothly migrated with adequate notice, and users can contact the Dimagi team for assistance during this transition.
```

Comment:
Nit: Would it make sense to include the "and users can contact the Dimagi team for assistance during this transition." part? This gives me vibes that the user might  be expected to do something on their end

Reply:
great point-- I added that because part of the plan had "Users can get help and support from the Dimagi team." as a key message.

Happy to remove, but I temporary added "with any questions" if that helps